packages/central-server/src/apiV2/import/importSurveys/constructQuestionValidators.js

```diff
@@ -232,4 +232,14 @@ export const constructQuestionValidators = models => ({
       return true;
     },
   ],
+  config: [
+    (cell, row) => {
+      if (row.type === 'CodeGenerator' && isEmpty(cell)) {
```

Comment:
```suggestion
      if (row.type === ANSWER_TYPES.CODE_GENERATOR && isEmpty(cell)) {
```

Prefer to use constant variable.

Reply:
Thanks for the suggestion, will do!